content/applications/inventory_and_mrp/purchase/advanced/procurement_expenses_report.rst

```diff
@@ -0,0 +1,126 @@
+
+.. tip::
+   While the |per| can also be :ref:`viewed <purchase/view-results>` as a :icon:`fa-bar-chart`
+   :guilabel:`(bar chart)`, :icon:`fa-line-chart` :guilabel:`(line chart)`, or :icon:`fa-pie-chart`
+   :guilabel:`(pie chart)`, the pivot view provides the most detailed view of the data, and is the
+   recommended starting point.
+
+Remove any default filters from the :guilabel:`Search...` bar. Then, click the |caret| to open the
+drop-down menu that contains the :guilabel:`Filters`, :guilabel:`Group By`, and
+:guilabel:`Favorites` columns.
+
+.. note::
+   Unless otherwise specified, the report displays data from both |RFQs| and |POs|. This can be
+   changed by selecting either :guilabel:`Requests for Quotation` or :guilabel:`Purchase Orders`
+   under the :guilabel:`Filters` column.
+
+Under the :guilabel:`Filters` column, select a date range to use for comparison. The report can be
+filtered by either :guilabel:`Order Date` or :guilabel:`Confirmation Date`. Choose one from the
+list, and click the |caret| to specify the date range, either by month, quarter, or year.
+
+Next, under the :guilabel:`Group by` column, select :guilabel:`Vendor`. Then, select
+:guilabel:`Product Category`, which is also located in the :guilabel:`Group By` column.
+
+.. note::
+   The selections under the :guilabel:`Group By` heading can be altered depending on the needs of
+   the individual company. For example, selecting :guilabel:`Product` instead of :guilabel:`Product
+   Category` provides a more in depth look at the performance of specific items, instead of an
+   entire category.
+
+Next, make a selection under the :guilabel:`Comparison` heading. These options are only available
+after the date range is selected under the :guilabel:`Filters` column, and vary based on that range.
+:guilabel:`Previous Period` adds a comparison to the previous period, such as the last month or
+quarter. :guilabel:`Previous Year` compares the same time period from the previous year.
+
+.. note::
+   While multiple time-based filters can be added at once, only one comparison can be selected at a
+   time.
+
+.. figure:: procurement_expenses_report/filters-groups.png
+   :align: center
+   :alt: The drop-down menu of filters, group by and comparison options for the procurement expenses
+         report.
+
+   The filter for Q2, comparison for **Previous Period**, and group-by for **Vendor** and **Product
+   Category** were selected.
+
+Add measures
+------------
+
+After selecting the :guilabel:`Filters`, :guilabel:`Group by`, and :guilabel:`Comparison` settings,
+click out of the drop-down menu.
+
+By default, the report displays with the following measures: :guilabel:`Order`, :guilabel:`Total`,
+:guilabel:`Untaxed Total`, and :guilabel:`Count`. Click :guilabel:`Measures` at the top-left to open
+the drop-down list of available measures.
+
+Click the following measures to include additional columns in the report:
```

Comment:
```suggestion
Click the following specific measures to include additional columns for the procurement expenses report:
```
just for added specificity; so you don't have to go through each option in the 'Measures' drop-down menu